tests/test_utils.py

```diff
@@ -164,5 +166,69 @@ def testTypeNames(self):
             self.assertEqual(getFullTypeName(item), typeName)
 
 
+class FindFilesTestCase(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
I think rather than setting up a temp directory and the like, you can just ask it to look in `tests/config` for `\.yaml\b` files and check that you find some representative ones.

Reply:
Is there overhead in tempdir that you're worried about? IMO it's better to do it this way & to know exactly what is and isn't expected, and to be able to test for that. And also better to not depend on a resource where it's not obvious that changing that will affect this test. But if you really want I'll change it(?)

Reply:
It's more code to support when the purpose of the test is to find files and not set up test infrastructure. It will be a bit faster to not create the temp dir but we create loads of them anyhow. I do understand that this way means you have full control over the results, but the other way doesn't have to be fragile if you are only looking for a subset of files rather than an exact list of every file and if we were to remove some of those config files there'd be breakage in many places. I'm not going to insist.

Reply:
I was able to change the unit test, getting expected lists of files using glob. I'm not sure it's really better, but it works and eliminates the temp file infrastructure.